### ISSUE TITLE
Output to the console when successfully loading .env file for servers and the CLI

### ISSUE THREAD

**Issue by elpete** [elpete/commandbox-dotenv#1]
To help users know when their `.env` file has been loaded, add some console output to the two different interceptors on successfully loading a file.  If no `.env` file is present, then nothing should be logged.

**Comment by bdw429s** [comment]
To ensure the logging messages get property output through the new interactive job API, inject the console logger and use it for any output messages.  CommandBox will take care of the rest, and this will also be CommandBox 4 compatible.

```
property name='consoleLogger' inject='logbox:logger:console';
```
```
consoleLogger.info( 'foo' );
consoleLogger.warn( 'foo' );
consoleLogger.debug( 'foo' );
consoleLogger.error( 'foo' );
```

**Comment by jsteinshouer** [comment]
I thought I would take a crack at this one if it is still needed.  I noticed that the console output is hidden by the interactive job API in CommandBox 4 once the job to start the server finishes.

So this would display but not really long enough to see it.

```
CommandBox:commandbox-dotenv-test> server start
 - | Starting Server
   |------------------------------
   | Java properties loaded from C:\Users\jason\Dropbox\dev\commandbox-dotenv-test\.env
   | WAR/zip archive already installed.
   | Server JVM arguments loaded from C:\Users\jason\Dropbox\dev\commandbox-dotenv-test\.env
   |------------------------------
```

Then you would see this.

```
CommandBox:commandbox-dotenv-test> server start
 ✓ | Starting Server
```

Just wanted to check and see if this is a concern? If so, I could possibly use the job API to create a nested job around loading the .env properties but would need some logic to determine if it exists since it is new in CommandBox 4. What are your thoughts?

**Comment by elpete** [comment]
Do the messages stay when using the `—verbose` flag? If so, I think this is fine. 

The other use case would be to output a message when loading CommandBox for the first time or on a reload of the shell. (Should be the same interception point.) That message should not be swallowed up.

**Comment by jsteinshouer** [comment]
It looks like the messages stay using the `--debug` flag. I will submit a PR for this.

**Comment by bdw429s** [comment]
Correct, if the user wants to see all the nitty gritty details, they can start with the `--debug` flag.  It would be pretty easy to add a nested interactive job so it stayed in the output nested under the server start.  Just a couple things to think about:
* CommandBox 3 doesn't have nested jobs so you'd need to detect their existence.  See how I do it here:
https://github.com/bdw429s/commandbox-fusionreactor/blob/master/ModuleConfig.cfc#L30
* To detect if there is an active job (assuming you're on CommandBox 4+) you would run `job.isActive()`

**Comment by elpete** [comment]
Resolved by #8
